float64 Theta1
float64 Theta2
float64 Theta3 
int8[4] Legs